vdev/src/commands/compile_vrl_wasm.rs

```diff
@@ -0,0 +1,25 @@
+use std::path::Path;
```

Comment:
On second review, I think this might be better to put under the `generate` subcommand, ie `vdev generate vrl-wasm`.